STM32F405RGTx/STM32F405RGTx.ioc

```diff
@@ -120,6 +129,7 @@ Mcu.UserName=STM32F405RGTx
 MxCube.Version=6.4.0
 MxDb.Version=DB.6.0.40
 NVIC.BusFault_IRQn=true\:0\:0\:false\:false\:true\:false\:false\:false
+NVIC.DMA2_Stream0_IRQn=true\:5\:0\:false\:false\:true\:true\:false\:true
```

Comment:
Are you trying to set the DMA interrupt on a higher priority here or lower priority? A lower number results in a higher priority so in this case you're decreasing priority, i.e. everything else will run before the dma interrupt

Reply:
I am trying to set it to a higher priority here
The DMA priority was set to "very high" through  cubeide and I noticed that bms also has it set to this as well despite the .ioc in cube ide showing "high priority"

Should I change the 5 to a 0 instead?

Reply:
Hmm, idk maybe this is a question for Ryan. For now you can leave it tho since I doubt we'll have timing issues